stumpy/core.py

```diff
@@ -471,19 +469,27 @@ def mass(Q, T, M_T=None, Σ_T=None):
 
     Q = np.asarray(Q)
     check_dtype(Q)
-    check_nan(Q)
```

Comment:
Should we remove the `check_nan` function if it isn't being used? I guess it is still being used in `gpu_stump` so, perhaps, we should create a Github issue to add the `np.nan`/`np.inf` stuff to `gpu_stump` and then delete the function afterward

Reply:
Yes, as long as it is being used we should keep in in core

Reply:
Added this issue for future tracking https://github.com/TDAmeritrade/stumpy/issues/125